### ISSUE TITLE
Prevent WebView from displaying “web page not available”

### ISSUE THREAD

**Issue by eros2187** [apache/cordova-plugin-inappbrowser#411]
I have an app that makes extensive use of a WebView. When the user of this app does not have an Internet connection, a page appears that says "web page not available" and various other texts. Is there a way not to show this generic text in my WebView? I would like to provide my error handling.

how can i solve you have solutions?

**Comment by tkallim** [comment]
same issue with codova 8.1.1, any update?

**Comment by eros2187** [comment]
yes any update status could be avoided by modifying the java code. but I'm not an expert on this.
